### PR TITLE
Added simple counter

### PR DESCRIPTION
**Pull request by dth99**
Added Simple counter program in Web Hacks

### CODE REVIEW

File: Web Hacks/simple counter/script.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const incr = document.querySelector('.incr');
+const decr = document.querySelector('.decr');
+
+let count = 0;
+incr.addEventListener("click", () => {
+    count++;
+    counter.innerHTML = count;
+})
+decr.addEventListener("click", () => {
+    if(count<=0){
```
Comment (bislara) [inline Web Hacks/simple counter/script.js:11]: This is a wonderful beginner project. The CSS looks so cool @dth99 

I have a doubt here. On clicking decrement, if the number is equal less than or equal to 0, why do we need to increase the value?  If you don't want to implement negative numbers then you could keep it equals to 0. 

Just let me know about this, I will merge soon after it.

**Review by bislara — APPROVED** [review]

File: Web Hacks/simple counter/script.js (hunk 1 above)
Comment (dth99) [reply Web Hacks/simple counter/script.js:11]: Yes sir, you are right.

Comment (bislara) [reply Web Hacks/simple counter/script.js:11]: Sorry I couldnt get you. Why do we need to increase the value of count?

Comment (dth99) [reply Web Hacks/simple counter/script.js:11]: Sir, please check now  logic is correct.

Comment (bislara) [reply Web Hacks/simple counter/script.js:11]: Hey bro, this is all I meant to say. 

`if(count<=0){
	count=0;
}
else
	count--;
`

Comment (dth99) [reply Web Hacks/simple counter/script.js]: I get it, you mean only for +ve value.
Now check big bro

**Comment by bislara** [comment]
@all-contributors please add @dth99 for code, doc

**Comment by dth99** [comment]
> @bislara
> 
> I've put up [a pull request](https://github.com/bislara/code-n-stitch/pull/243) to add @dth99! tada

thank you sir

**Pull request merged**